Index: rrootage-0.23a/src/screen.c
===================================================================
--- rrootage-0.23a.orig/src/screen.c	2007-08-29 19:50:14.000000000 +0000
+++ rrootage-0.23a/src/screen.c	2007-08-29 19:50:14.000000000 +0000
@@ -128,7 +128,7 @@
 static GLuint titleTexture;
 #define TITLE_BMP "title.bmp"
 
-int resolution = MEDIUM_RESOLUTION;
+int resolution = HIGH_RESOLUTION;
 int windowMode = 0;
 int brightness = DEFAULT_BRIGHTNESS;
 Uint8 *keys;
